### ISSUE TITLE
suggested fix to comment on losses histogram of the pre-trained model.

### ISSUE THREAD

**Issue by AmrKhalifa** [unlearning-challenge/starting-kit#2]
The comment after the plot of histogram of losses on train vs test set of the pre-trained model (first plot) 
says: 
"As per the above plot, the distributions of losses are quite different between the _**forget and retain set**_. This suggests that the simple MIA that we're considering should be reasonably effective."

I believe this should be changed to: 
"As per the above plot, the distributions of losses are quite different between the **_train and test set_**. This suggests that the simple MIA that we're considering should be reasonably effective."

**Comment by eleniTriantafillou** [comment]
Indeed, great catch Amr, that was a typo. I've updated the notebook
accordingly, and added some further clarifications :)
